corehq/apps/data_dictionary/views.py

```diff
@@ -159,6 +159,7 @@ def main_context(self):
                 couch_user=self.request.couch_user,
                 project=self.request.project
             ),
+            'question_types': [{'value': k, 'display': v} for k, v in PROPERTY_TYPE_CHOICES if k],
```

Comment:
I wanted to exclude the empty value, because the default placeholder for the JS is different than the default for the model itself. (One of them makes sense as a placeholder for a form, one makes sense in the exports)